apim-operator/deploy/scenarios/README.md

```diff
@@ -0,0 +1,45 @@
+# k8s-apim-operator Scenarios
+
+## Scenario1
+
+> ##### This scenario demonstrates ratelimiting (throttling), OAuth2 flow, Analytics, Security kind, API addition, update, show and delete
+
+- Follow the main README and deploy the apim-operator.
+ 
+##### Navigate to the k8s-apim-operator/apim-operators/scenarios/scenario1 directory and execute the following command
+
+- Deploy apim and analytics configurations
+    - ***kubectl apply -f apim_analytics_configs.yaml***
+    - Note: uploading cycle of usage files to analytics server is set for a shorter period (1 minute) for demonstration purposes
+
+- Create 2 namespaces <br /> 
+    - ***kubectl apply -f namespace.yaml***
+
+- Create few ratelimiting instances in the abobe namespaces. <br /> 
+    - ***kubectl apply -f ratelimiting_instances.yaml***
+    - Note: Samples are given for all 3 types of ratelimiting kind: advance, subscription, application. Using advance3 and advance5 - two advance ratelimiting instances is sufficient to demonstrate this scenario. To work with application and subscription ratelimiting types, user need to have created those policies in the APIM server
+
+- Create OAuth2 type instances of security kind and secrets with credentials for the APIM server. <br />
+    - ***kubectl apply -f oauth_security.yaml***
+    - Note: User must deploy two secrets with the certificates. The following command can be used to a create secret from the certificate pem file.
+    - ***kubectl create secret generic wso2am260-secret --from-file=wso2am260.pem -n=wso2-test-ns1***
+    - ***kubectl create secret generic wso2am260-secret --from-file=wso2am260.pem -n=wso2-test-ns2***
+
+- Do hostmapping for APIM and analytics servers.  <br />
+    - ***kubectl apply -f host_mapping.yaml***
+
+- Deploy two APIs in the created namespaces.  <br />
+    - ***kubectl add api myapi1 --from-file=swagger_ns1_oauth.yaml -n=wso2-test-ns1***
```

Comment:
We should not use wso2-system namespace to create any API.

Reply:
Or any kind.

Reply:
wso2-system namespace was not used for api creation, but the test namespaces created in the 1st step(wso2-test-ns1 and wso2-test-ns2) were used.